config/packages/github_api.yaml

```diff
@@ -3,7 +3,7 @@ services:
         arguments:
             - '@Github\HttpClient\Builder'
         calls:
-            - ['authenticate', ['%env(GITHUB_TOKEN)%', 'http_token']]
+            - ['authenticate', ['%env(GITHUB_TOKEN)%', null, 'access_token_header']]
```

Comment:
Note, this will break production. 

The `carsonbot` user needs to login to github> Go to settings > developers settings > personal access token (link: https://github.com/settings/tokens) and then generate a new access token with "api" privileges.

Reply:
Can you explain this a bit more? carsonbot already has a personal access token, with with just one scope - `public_repo`. What difference does this change make? And how would this new API token be different? I don't see anything specific to "api" privileges when I create a token.

Reply:
I am wrong. 

The `http_token` and `access_token_header` is the same. But the former is a deprecated value. 
https://github.com/KnpLabs/php-github-api/blob/2.x/lib/Github/HttpClient/Plugin/Authentication.php#L54-L58

No bc break, this change is fine.